packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -177,7 +184,7 @@ The following customizations aren't supported:
 * **Okta-hosted widget:** The **Help title** link in the **Customized Help Links** section of the Customization page has been removed and isn't supported.
 * **Self-hosted widget:** The **Need help signing in** string has been removed and isn't supported.
 
-See [Okta Sign-in Widget Help](https://github.com/okta/okta-signin-widget/#help-links).
+See [Okta Sign-in Widget: Help links](https://github.com/okta/okta-signin-widget/#help-links).
```

Comment:
Links is capitalized in the xref you go to, but I recognize it's not our style.

Reply:
I'm leaving Links uncapitalized here.